platform/event/event_descriptor.proto

```diff
@@ -0,0 +1,64 @@
+/**
+ * # Event Descriptor
+ * Unique identifier for an event.
+ *
+ * ### Keywords
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119).
+ */
+syntax = "proto3";
+
+package com.hedera.hapi.platform.event;
+
+/*
+ * Hedera Network Services Protobuf
+ *
+ * Copyright (C) 2018 - 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+option java_package = "com.hedera.hapi.platform.event";
+// <<<pbj.java_package = "com.hedera.hapi.platform.event">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * Unique identifier for an event.
```

Comment:
The `EventDescriptor` (as currently defined) cannot be part of the gossip event, since the `EventDescriptor` contains the event hash.

Including the event hash in the gossip event would be a waste, since every node must independently generate that hash anyway.

Reply:
Perhaps detail that in the file comment. A few sentences about the design and where event hashes are _expected_ to be and why things are structured the way they are.

The file comment is a great place for larger blocks of text that discuss design decisions and document the use or purpose for various messages. The file comment may also include multiple sections with headings and other text styles not necessarily appropriate for individual field descriptions.